Src/Newtonsoft.Json/Bson/BsonReader.cs

```diff
@@ -175,7 +175,7 @@ private string ReadElement()
         }
 
         /// <summary>
-        /// Reads the next JSON token from the stream.
+        /// Reads the next JSON token from the underlying <see cref="BinaryReader"/> or <see cref="Stream"/>.
```

Comment:
Please confirm this again. Internally, `BsonReader` always uses a `BinaryReader`. But outwardly, things look different: A user of the `BsonReader` class sees two public constructors, one accepting a `Stream`, the other accepting a `BinaryReader`. So there's a choice, and I tried to keep the comments in line with that.

But if you say so, I can change these comments to refer only to one or the other. (In that case, I'd suggest choosing `BinaryReader`.)